src/Vixen.Modules/Editor/TimedSequenceEditor/TimedSequenceEditorForm.cs

```diff
@@ -84,7 +83,7 @@ public partial class TimedSequenceEditorForm : BaseForm, IEditorUserInterface, I
 
 		private readonly Timer _autoSaveTimer = new Timer();
 
-		// a mapping of effects in the sequence to the element that represent them in the grid.
+		// a mapping of effects in the sequence to the effectDetail that represent them in the grid.

```

Comment:
I am not sure I understand all these renaming of effect / element to effectDetail. Is this some sort of refactor mistake?